src/spec-lang/tc/builtins.ts

```diff
@@ -212,3 +213,30 @@ export const BuiltinAddressMembers = new Map<string, TypeNode | [TypeNode, strin
     ],
     ["code", [new PointerType(new BytesType(), DataLocation.Memory), ">=0.8.0"]]
 ]);
+
+export const ContractTypeMembers = new BuiltinStructType(
+    "<type(contract)>",
+    new Map([
+        ["name", new PointerType(new StringType(), DataLocation.Memory)],
+        ["creationCode", new PointerType(new BytesType(), DataLocation.Memory)],
+        ["runtimeCode", new PointerType(new BytesType(), DataLocation.Memory)]
+    ])
+);
+
+export const InterfaceTypeMembers = new BuiltinStructType(
+    "<type(interface)>",
+    new Map<string, TypeNode>([
+        ["name", new PointerType(new StringType(), DataLocation.Memory)],
+        ["interfaceId", new FixedBytesType(4)]
```

Comment:
Available since [Solidity v0.6.7](https://github.com/ethereum/solidity/releases/tag/v0.6.7). I guess it is worth to specify minimal required version.

Reply:
Good catch. Fixed in Fixed in 9555f70.